app/scripts/views/confirm.js

```diff
@@ -36,7 +36,6 @@ define(function (require, exports, module) {
       // ephemeral properties like unwrapBKey and keyFetchToken
       // that need to be sent to the browser.
       this._account = this.user.initAccount(this.model.get('account'));
-      this.flow = this.model.get('flow');
```

Comment:
Removals like this are awesome, there were times I scratched my head and asked myself "What's this here for?"